nova_vm/Cargo.toml

```diff
@@ -24,16 +24,26 @@ unicode-normalization = { workspace = true }
 wtf8 = { workspace = true }
 
 [features]
-default = ["math", "json", "date", "array-buffer", "shared-array-buffer", "weak-refs", "atomics"]
+default = [
+    "math",
+    "json",
+    "date",
+    "array-buffer",
+    "shared-array-buffer",
+    "weak-refs",
+    "atomics",
+    "regexp"
+]
 array-buffer = []
 atomics = ["array-buffer", "shared-array-buffer"]
 date = []
 interleaved-gc = []
 json = ["sonic-rs"]
 math = []
 shared-array-buffer = []
-typescript = []
 weak-refs = []
+regexp = []
+typescript = []
```

Comment:
**question:** Is there a reason to keep typescript at the bottom?

I sorted these into alphabetical order (except for the default features) the other week.

Reply:
lmao no reason didnt realize you did that, was a slight nitpick mentally since its the only one not used by default

Reply:
would you prefer i change it back? might make more sense

Reply:
Regexp at least should be alphabetical. TS can be last as a "non-default feature"